packages/identity-app/src/Identity.tsx

```diff
@@ -2,42 +2,134 @@
 // This software may be modified and distributed under the terms
 // of the Apache-2.0 license. See the LICENSE file for details.
 
-import { Container, Grid } from '@substrate/ui-components';
+import { Grid } from '@substrate/ui-components';
 
 import React from 'react';
 import { RouteComponentProps } from 'react-router-dom';
-import Tab from 'semantic-ui-react/dist/commonjs/modules/Tab';
+import Dropdown, { DropdownProps } from 'semantic-ui-react/dist/commonjs/modules/Dropdown';
+import Menu from 'semantic-ui-react/dist/commonjs/collections/Menu';
 
-import { AddressBook } from './AddressBook';
-import { Create } from './Create';
 import { SavedAccounts } from './SavedAccounts';
-import { Wallet } from './Wallet';
+import { SavedAddresses } from './SavedAddresses';
+import { ManageAccounts } from './ManageAccounts';
+import { ManageAddresses } from './ManageAddresses';
+import { IdentityManagementScreen, MatchParams } from './types';
 
-interface Props extends RouteComponentProps {
-  basePath: string;
-}
+interface Props extends RouteComponentProps<MatchParams> {}
+
+type State = {
+  menuOption: 'Accounts' | 'Addresses',
+  screen: IdentityManagementScreen
```

Comment:
tbh react-router-dom is basically created to avoid having a `screen` field in state. Plus I think having different routes for edit/create/restore makes sense.